aztec/src/main/kotlin/org/wordpress/aztec/source/Format.kt

```diff
@@ -6,22 +6,27 @@ import java.util.regex.Pattern
 object Format {
 
     // list of block elements
-    private val block = "div|span|br|blockquote|ul|ol"
+    private val block = "div|span|br|blockquote|ul|ol|li|h1|h2|h3|h4|h5|h6"
 
     fun addFormatting(content: String): String {
-        // let's use Jsoup for HTML formatting for now because it works out of the box
         val doc = Jsoup.parseBodyFragment(content)
-        return doc.body().html()
+
+        //remove newline around all non block elements
```

Comment:
Oh I see. There is a bunch of stuff that I converted but have no idea if they're really necessary. The thing is some of the replacements were solely because the hybrid editor did not support certain tags/features. I only learned about that after the conversion, so I'm not sure we want to just blindly use these.. For example, the `<p>` tags get removed for some reason, which we probably don't want.